proxy/server.js

```diff
@@ -16,8 +16,11 @@ app.get("/nxt", async function (req, res) {
       "cache-control": "no-cache",
       pragma: "no-cache",
       "upgrade-insecure-requests": "1",
+      "Access-Control-Allow-Origin": "*",
```

Comment:
the reason this doesn't work is that CORS is set by the server. here you're trying to set it in a request (which is being sent to the nodes.jup.io api).

no action here, just explaining why this wouldn't work.